docs/getting-started.mdx

```diff
@@ -8,46 +8,43 @@ import { AssetsProvider } from '@site/src/components/assets-context';
 import DownloadCard from '@site/src/components/download-card';
 import DownloadCardStyles from '@site/src/css/download-card.module.css';
 
+## Share your network resources within minutes
+
+Easily share private network resources with public internet users, securely, without having to alter any of your local network configuration.
+You can share resource using [NetFoundry's cloud-hosted zrok](https://myzrok.io/),
+or follow our [self-hosted guide](/docs/category/self-hosting/) to manage your own instance.
+
+* Download the binary for your platform [here](https://github.com/openziti/zrok/releases/latest)
+* Sign up for a myzrok.io account [here](https://myzrok.io/) to enable your personal share token
+* Run `zrok enable <your_token>` to enable your shell environment for sharing
+* Run `zrok share public localhost:8080` to start your first share!
+
+![zrok share public](images/zrok_share_public.png)
+
+Access your share from anywhere!
+
+```
+$ curl -s https://dslno640nct4.share.zrok.io | head
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1"/>
+    <meta name="theme-color" content="#000000"/>
+    <meta name="description" content="zrok ui"/>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+```
+
+zrok provides several different share types, including:
+
+* [public shares](./concepts/sharing-public.mdx) for [web services](./concepts/http.md) or [files](./concepts/files.md)
+* [private shares for web services or files](./concepts/sharing-private.mdx)
+* [reserved shares](./concepts/sharing-reserved.md)
+* [tcpTunnel shares](./concepts/tunnels.md)
+* [udpTunnel shares](./concepts/tunnels.md)
+* [drives](./guides/drives.mdx)
+* [vpn](./guides/vpn/vpn.md)
 
-## Get an Account
```

Comment:
I feel attached to the two choice cards for getting started. Getting an account seems like the correct first step. I added them because there wasn't a clear path to myzrok.io.

Reply:
I'll play with this a little bit more and shift the "sign up" instruction to be first in the list instead of second.